baseband/gsb/tests/test_gsb.py

```diff
@@ -229,7 +229,7 @@ def test_payload(self):
             assert payload1.data.shape == (8192, 1)
             assert payload1.sample_shape == (1,)
             assert payload1.sample_shape.nchan == 1
-            # Passing ``None`` to `payloadsize`throws an error.
+            # Passing `None` to ``payloadsize`` throws an error.
```

Comment:
This is actually a rather useless comment! If we keep anything: `Reader needs to be passed an actual payloadsize.`